src/components/ui/JobsPageProTipCard.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+/* global React$Node */
+
+import React from 'react'
+
+import theme from '../../styles/theme'
+import typo from '../../styles/typo'
+
+const cmz = require('cmz')
+
+const cx = {
+  card: cmz(
+    typo.regularText,
+    `
+      & {
+        background: ${theme.baseYellow}
+        border: 1px solid ${theme.lineYellow}
+        box-shadow: 4px 4px 0px ${theme.baseYellowDark.fade(0.85)}
+        padding: 24px
+        line-height: 1.4
+        font-weight: 400
+      }
+
+      & a {
+        color: ${theme.typoAnchor}
+      }
+
+      & a:hover {
+        color: ${theme.typoAnchorHover}
+      }
+
+      &:not(:only-child) {
+        margin: 0 0 38px
+      }
+    `
+  ),
+
+  heading: cmz(`
+    & {
+      font-size: 18px
+      line-height: 1.4
+      margin: 0
+    }
+
+    &:not(:only-child) {
+      margin: 0 0 12px
+    }
+  `)
+}
+
+type Props = {
+  children?: React$Node,
+  heading?: string
+}
+
+const JobsPageProTipCard = ({ children, heading }: Props) => children || heading ? (
+  <div className={cx.card}>
+    {heading && (
```

Comment:
If my prev comment is applied this check isn't needed at all so we can shorten the component rendering code.
```suggestion
    <h2 className={cx.heading}>{heading}</h2>
```

Reply:
I think this will not behave as expected. If I made this change the component will not be rendered if `heading` is not available.

This is what is expected with current code:

<img width="974" alt="Screen Shot 2019-11-15 at 11 33 13" src="https://user-images.githubusercontent.com/131859/68950885-d1d5da80-079b-11ea-843c-35c041f5bb9c.png">